### ISSUE TITLE
LSMOL on nuScenes

### ISSUE THREAD

**Issue by TedLentsch** [Robertwyq/LSMOL#5]
I would like to use LSMOL on nuScenes. However, according to the catalog in the README, the code is not published fully yet, i.e. the 3D instance segmentation code is missing is stated. Is this still true, or is it already released?

Btw, what does LSMOL stand for? I have been searching in the paper, but I have not found the meaning of the abbreviation. L... S... M... Object Localization?

**Comment by Robertwyq** [comment]
@TedLentsch 
Indeed, the code for 3D instance segmentation has not been released yet. Since most of the code was written specifically for Waymo, you can initially try estimating the scene flow on the nuScenes dataset. (using NSFP or FastNSF) The name "LSMOL" originates from the former name of this project, which was "LiDAR Supervised Moving Objects Localization."

**Comment by TedLentsch** [comment]
@Robertwyq thank you for your reply! I will try to get it to work on nuScenes.
